src/test/rustdoc-ui/issue-102986.rs

```diff
@@ -0,0 +1,4 @@
+struct Struct {
+    y: (typeof("hey"),),
+    //~^ `typeof` is a reserved keyword but unimplemented
```

Comment:
For future reference, I think it's conventional to do `//~^ ERROR` just to make sure we're asserting the kind of diagnostic here

Reply:
ooops, I forgor 💀